lib/types/src/types/core/message.d.ts

```
@@ -0,0 +1,91 @@
+        action: Action.CantDo;
+    };
+} | {
+    type: 'rate';
+    message: MessageKind & {
+        action: RatingActions;
+    };
+};
+type OrderActions = Action.NewOrder | Action.TakeSell | Action.TakeBuy | Action.PayInvoice | Action.AddInvoice | Action.FiatSent | Action.FiatSentOk | Action.Release | Action.Released | Action.Cancel | Action.Canceled | Action.WaitingBuyerInvoice | Action.WaitingSellerToPay | Action.BuyerTookOrder | Action.HoldInvoicePaymentAccepted | Action.HoldInvoicePaymentSettled | Action.HoldInvoicePaymentCanceled | Action.CooperativeCancelInitiatedByYou | Action.CooperativeCancelInitiatedByPeer | Action.CooperativeCancelAccepted;
+type DisputeActions = Action.Dispute | Action.DisputeInitiatedByYou | Action.DisputeInitiatedByPeer;
+type RatingActions = Action.Rate | Action.RateUser | Action.RateReceived;
+export {};
```

Biome

[error] This empty export is useless because there's another export or import.

This import makes useless the empty export.

Safe fix: Remove this useless empty export.


(lint/complexity/noUselessEmptyExport)